Codegen/src/Nodes/TypedDataNode.cs

```diff
@@ -1002,10 +1006,10 @@ public static List<ITextLine> InsertChildren(ColumnsLayout? layout, List<string>
                         CopyDirective copy = child.CodeElement.FirstCopyDirective;
                         //The first data coming from a copy is used to recover the Clause COPY, the other would be only a repetition of this one, so we skip them.
                         //Even with the same name, two different Clause COPY are differentiated by their token lines
-                        if (usedCopies.Contains(copy) == false)
+                        if (usedCopy == null || usedCopy.Equals(copy) == false)
```

Comment:
Why use `Equals` here ? `CopyDirective` does not define custom equality comparison mechanism so reference comparison is fine.

Reply:
using ==; done.
https://github.com/TypeCobolTeam/TypeCobol/pull/1646/commits/5ec741f218377464909aae42e8454ea4de4d1d2e